app/views/single_pages/sample_upload_content.html.erb

```diff
@@ -0,0 +1,405 @@
+<%
```

Comment:
there is a lot of code directly in the single view here, and would be easier to understand if split into partials and/or view helper methods